docs/platforms/python/configuration/options.mdx

```diff
@@ -172,6 +172,8 @@ This parameter controls whether integrations should capture HTTP request bodies.
 - `medium`: Medium and small requests will be captured (typically 10KB).
 - `always`: The SDK will always capture the request body as long as Sentry can make sense of it.
 
+Please note that the Sentry server [limits HTTP request body size](https://develop.sentry.dev/sdk/data-handling/#variable-size). The server **always enforces its size limit**, regardless of how you configure this option!
```

Comment:
Not sure if we need the bold text and the exclamation point; feels like it clashes with the usual tone of the docs a bit. Wdyt about simply leaving them out?

```suggestion
Please note that the Sentry server [limits HTTP request body size](https://develop.sentry.dev/sdk/data-handling/#variable-size). The server always enforces its size limit, regardless of how you configure this option.
```